zipline/assets/assets.py

```diff
@@ -26,112 +25,89 @@
 
 from zipline.errors import (
     ConsumeAssetMetaDataError,
-    InvalidAssetType,
     MultipleSymbolsFound,
     RootSymbolNotFound,
-    SidAssignmentError,
     SidNotFound,
     SymbolNotFound,
     MapAssetIdentifierIndexError,
 )
-from zipline.assets._assets import (
+from zipline.assets import (
     Asset, Equity, Future
 )
+from abc import ABCMeta
```

Comment:
Can we move this to the top of the imports?

imports should be in 3 groups separated by 1 line and sorted alphabetically internally:

1. stdlib
1. third party packages
1. packages defined in this project